grails-app/migrations/changesets/changelog-11.0.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="vvicario" id="changelog-11.0-1">
+        <addColumn tableName="consent_collection_link">
+            <column name="pii" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="compliance" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="sharing_type" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="text_sharing_type" type="LONGTEXT"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="text_compliance" type="LONGTEXT"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="international_cohorts" type="LONGTEXT"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="require_mta" type="VARCHAR(255)"/>
+        </addColumn>
+    </changeSet>
+
+    <changeSet author="vvicario" id="changelog-11.0-2">
+        <addColumn tableName="storage_document">
+            <column name="consent_key" type="VARCHAR(255)"/>
+        </addColumn>
+    </changeSet>
+
+    <changeSet author="vvicario" id="changelog-11.0-3">
+        <dropNotNullConstraint columnName="project_key"
+                               columnDataType="VARCHAR(255)"
+                               tableName="storage_document"/>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
One thing this changeset does **not** do, and probably should, is add a nullable column for the id of consent collection link. I may be misunderstanding things a bit - which could be a naming problem.

Reply:
Agree, we missunderstood, working on change it, I'll let you know when changes are done. Thanks